recipes/chempy/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "chempy" %}
+{% set version = "0.8.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 565fe05a66aafce322c28ef4a302de3187709f7238a194fee00c8df7e7c5b690
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - matplotlib-base >=2.2.5
+    - notebook >=5.7.8
+    - numpy >=1.16.4
+    - python >=3.7
+    - scipy >=1.2.3
+    - sym >=0.3.4
+    - sympy >=1.5.1
+    - pytest
```

Comment:
Is pytest really a runtime dependency? It's fine if it is, just unusual

Reply:
It is, perhaps bad design to do a module level import of it in a util module, but here's the import:
https://github.com/bjodah/chempy/blob/ea9b8766f26cecd363ef3293b0c4d0c6ae357063/chempy/util/testing.py#L8